assets/js/modules/adsense/components/dashboard/DashboardTopEarningPagesWidget.js

```diff
@@ -180,7 +180,14 @@ function DashboardTopEarningPagesWidget( props ) {
 			primary: true,
 			Component: ( { row } ) => {
 				const [ title, url ] = row.dimensions;
-				return <Link href={ url } children={ title } external />;
+				return (
+					<Link
+						href={ url }
+						children={ title }
+						external
+						hideExternalIndicator
```

Comment:
See above - why was this changed, could we remove this prop again?

Reply:
@felixarntz See my comment here: https://github.com/google/site-kit-wp/pull/5314#discussion_r907369159